src/docs/agenda.md

```diff
@@ -1,15 +1,17 @@
 # Agenda
 
-Under development: TBD
+!!! note
+    This section is currently under development. The information provided is subject to change.
 
 ## Tutorial Structure
 
-The tutorial will occur over half a day and focuses on introducing some common open science principles and their usage within education technology, providing an example on using the *Open Science Framework* to create a project, post content, and preregister studies, and using previous papers to apply the learned principles and any additional reproduction mitigations.
```

Comment:
instead of "over half a day" let's say 3 hours
add "add contributors" to OSF details, people are always so confused